src/components/header/header-bar.vue

```diff
@@ -24,6 +24,21 @@ watch(
   },
   { immediate: true }
 )
+
+function onClick() {
+  if (!layersOpen.value) {
+    setLayersOpen(true)
+    myLayersTabOpen.value && setMyLayersTabOpen(false)
+    setThemeGridOpen(true)
+  } else if (layersOpen.value) {
+    if (themeGridOpen.value) {
+      setLayersOpen(false)
```

Comment:
As in v3, but more difficult to test here, as there are no other active footer buttons yet.